package.json

```diff
@@ -103,6 +104,7 @@
     "micromatch": "4.0.4",
     "module-alias": "2.2.2",
     "monaco-yaml": "3.2.1",
+    "mousetrap": "^1.6.5",
```

Comment:
we are using `react-hotkeys-hook` in the `HotKeysHandler` component, could you try using that for the shortcuts instead of adding `mousetrap` ?

Reply:
Sure